### PR TITLE
Vb 2060 get contact details and prison details using orchestration service

### PR DESCRIPTION
**Pull request by dhirajshettymoj**

### CODE REVIEW

**Comment by dhirajshettymoj** [comment]
Setting as a draft PR as needs to be merged post chat with the front end team.

**Review by AledWEvans — APPROVED** [review]
LGTM

**Review by AledWEvans — CHANGES_REQUESTED** [review]
please change

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/IntegrationTestBase.kt [hunk 1]
```diff
@@ -38,13 +41,17 @@ abstract class IntegrationTestBase {
     val visitSchedulerMockServer = VisitSchedulerMockServer(ObjectMapper().registerModule(JavaTimeModule()))
     val prisonApiMockServer = PrisonApiMockServer(ObjectMapper().registerModule(JavaTimeModule()))
     val prisonOffenderSearchMockServer = PrisonOffenderSearchMockServer(ObjectMapper().registerModule(JavaTimeModule()))
+    val prisonerContactRegistryMockServer = PrisonerContactRegistryMockServer(ObjectMapper().registerModule(JavaTimeModule()))
+    val prisonRegisterMockServer = PrisonRegisterMockServer(ObjectMapper().registerModule(JavaTimeModule()))
```
[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/IntegrationTestBase.kt:47]: @AledWEvans  - updated

[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/IntegrationTestBase.kt:47]: updated

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/PrisonerProfileClient.kt [hunk 2]
```diff
@@ -35,6 +46,58 @@ class PrisonerProfileClient(
...
+      }
+  }
+
+  private fun setVisitDetails(prisonerProfile: PrisonerProfileDto) {
+    val contactsMap = getContactsForPrisoner(prisonerProfile)
+    val prisonsMap = getAvailablePrisons()
+
+    prisonerProfile.visits.forEach { visit ->
+      contactsMap?.let { setVisitorNames(visit, contactsMap) }
+      prisonsMap?.let { setPrisonNames(visit, prisonsMap) }
```
[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/PrisonerProfileClient.kt]: Updated

**Pull request merged**